BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/Extensibility/Implementation/Tracing/SelfDiagnostics/SelfDiagnosticsConfigRefresherTest.cs

```diff
@@ -91,13 +140,25 @@ private static byte[] ReadFile(string filePath, int byteCount)
             }
         }
 
-        private static void CreateConfigFile()
+        private static void CreateConfigFile(bool userDefinedLogDirectory = true)
```

Comment:
Did you ever invoke this method with `false`?

Reply:
updated testcase